src/components/StakingVaults/EarnOpportunities.tsx

```diff
@@ -130,9 +145,28 @@ export const EarnOpportunities = ({ assetId, accountId }: EarnOpportunitiesProps
         </HStack>
       </CardHeader>
       {Boolean(allRows?.length) && (
-        <CardBody pt={0} px={4}>
-          <StakingTable data={allRows} onClick={handleClick} />
-        </CardBody>
+        <Tooltip
+          label={translate('defi.noAccountsOpportunities')}
+          isDisabled={shouldEnableOpportunities}
+        >
+          <CardBody
+            sx={
+              !shouldEnableOpportunities
```

Comment:
Can use `&&` syntax if chakra is happy with that, to avoid negative case on the left of the ternary

Reply:
stale review as everything changed on that part